src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.Move.cs

```diff
@@ -0,0 +1,220 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public class Directory_Move_Tests : FileSystemWatcherTest
+    {
+        #region WindowsTests
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public void Windows_Directory_Move_To_Same_Directory()
+        {
+            DirectoryMove_SameDirectory(WatcherChangeTypes.Renamed);
+        }
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public void Windows_Directory_Move_From_Watched_To_Unwatched()
+        {
+            DirectoryMove_FromWatchedToUnwatched(WatcherChangeTypes.Deleted);
+        }
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public void Windows_Directory_Move_To_Different_Watched_Directory()
+        {
+            DirectoryMove_DifferentWatchedDirectory(WatcherChangeTypes.Changed);
+        }
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public void Windows_Directory_Move_From_Unwatched_To_Watched()
+        {
+            DirectoryMove_FromUnwatchedToWatched(WatcherChangeTypes.Created);
+        }
+
+        [Theory]
+        [InlineData(false)]
+        [InlineData(false)]
+        [PlatformSpecific(PlatformID.Windows)]
+        public void Windows_Directory_Move_In_Nested_Directory(bool includeSubdirectories)
+        {
+            DirectoryMove_NestedDirectory(includeSubdirectories ? WatcherChangeTypes.Changed | WatcherChangeTypes.Renamed : 0, includeSubdirectories);
+        }
+
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)]
+        public void Windows_Directory_Move_With_Set_NotifyFilter()
+        {
+            DirectoryMove_WithNotifyFilter(WatcherChangeTypes.Renamed);
+        }
+
+        #endregion
+
+        #region UnixTests
+
+        [Fact]
+        [PlatformSpecific(PlatformID.AnyUnix)]
+        public void Unix_Directory_Move_To_Same_Directory()
+        {
+            DirectoryMove_SameDirectory(WatcherChangeTypes.Renamed);
```

Comment:
A bunch of these tests look identical to a corresponding one above for Windows, e.g. this one and https://github.com/dotnet/corefx/pull/8231/files#diff-a7023715f794619bd9b6428067ab4b1cR15.  Why are they distinct?